src/app/cart/cart.component.js

```diff
@@ -60,8 +60,17 @@ class CartController {
     this.$window.location.href = this.sessionService.getRole() === 'REGISTERED' ? 'checkout.html' : 'sign-in.html';
   }
 
-  donationStartDate( month, day ) {
-    return new Date( 0, Number( month ) - 1, Number( day ) );
+  donationStartDate( config ) {
```

Comment:
Once the recurring order is completed, we have an actual next-draw-date from siebel. I think this is just for new/none completed recurring items. I do agree, it would probably help to add it to the cartService so any orders already have it.